README.md

```diff
@@ -166,6 +180,20 @@ Delay line.
 	- `DelayTime`
 	- `DelayTimeModulation`
 
+### EQBPFilter
+
+Non-resonant, variable width bandpass filter.
+
+- Inputs: `Frequency`, `Bandwidth`, `FM`, `BandwidthModulation`
+- Outputs: `Out`
+- Parameters:
+	- `Rate`
+	- `Depth`
+	- `Smooth`
+	- `RateModulation`
+	- `DepthModulation`
+	- `SmoothModulation`
+
```

Comment:
I'd love to see this getting aligned namewise:
Rate -> FM -> RateModulation (idea: Rate -> RateModulation -> RateModulation or Frequency -> FM -> FM?)
Depth -> BandwidthModulation -> DepthModulation (Depth -> DepthModulation -> DepthModulation perhaps?)

Also: SmoothModulation appears not to be applicable right now?

Edit: Sorry. It appears Smooth input is missing?

Reply:
this is a copy and paste typo ! I believe the parameters are `Frequency`, `Bandwidth`, `FM`, and `BandwidthModulation`

Reply:
No problem. There's also SC helper methods for generating these doc and lua sections:
- `Rrrr.generateModuleDocs`
- `Engine_R.generateLuaSpecs`